threescale/http/client_test.go

```diff
@@ -895,6 +908,12 @@ func getUsageReportXML(t *testing.T) string {
          <max_value>4</max_value>
          <current_value>1</current_value>
       </usage_report>
+      <usage_report metric="hits" period="hour">
+         <period_start>2019-02-22 14:32:00 +0000</period_start>
+         <period_end>2019-02-22 15:32:00 +0000</period_end>
```

Comment:
Per-hour timestamps are guaranteed to be absolute, non-overlapping with each other, and starting at HH:00:00 and finishing at HH:59:59.
```suggestion
         <period_start>2019-02-22 14:00:00 +0000</period_start>
         <period_end>2019-02-22 14:59:59 +0000</period_end>
```

Reply:
(the being absolute and non-overlapping applies to all granularities, not just hour)